spanner/spanner_snippets/spanner/spanner_create_instance_config.go

```diff
@@ -0,0 +1,80 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package spanner
+
+// [START spanner_create_instance_config]
+
+import (
+	"context"
+	"fmt"
+	"io"
+
+	instance "cloud.google.com/go/spanner/admin/instance/apiv1"
+	instancepb "google.golang.org/genproto/googleapis/spanner/admin/instance/v1"
+)
+
+// createInstanceConfig creates a custom spanner instance config
+func createInstanceConfig(w io.Writer, projectID string, userConfigName, baseConfigID string) error {
+	// projectID = `projects/<project>`
+	// userConfigName = `custom-nam11`
+	// baseConfigID = `projects/<project>/instanceConfigs/nam11`
+	ctx := context.Background()
```

Comment:
I think we should have a context with timeout here and in other samples (5mins, maybe?), as we're waiting for long running operations to complete here.